### PR TITLE
Support minor release for release script

### PR DESCRIPTION
**Pull request by beeender**

### CODE REVIEW

**Comment by zaki50** [comment]
Could you remove a comment in line 5?
` # FIXME: Only patch release is supported now.`

File: tools/release.sh [hunk 1]
```diff
@@ -93,14 +93,35 @@ prepare_branch() {
...
+        while true
+        do
+            read -r -p "Type the command to edit CHANGELOG.md, default(vim):" editor
+            if [ -z "$editor" ] ; then
+                editor="vim"
+            fi
+            "$editor" CHANGELOG.md
+
+            read -r -p "Please merge the unreleased entries in the 'CHANGELOG.md' and then press any key to continue..." _
+            if [ "$(grep -c "YYYY-MM-DD" CHANGELOG.md)" -eq 1 ] ; then
```
Comment (zaki50) [inline tools/release.sh]: We can write just`if grep -q "YYYY-MM-DD" CHANGELOG.md; then` here.

Comment (beeender) [inline tools/release.sh]: this is to check if the changelog entries are merged correctly. eg.:
```
2.2.0 (YYYY-MM-DD)

...

2.1.3 (YYYY-MM-DD)
```

The 2.1.3 should be removed and merge to 2.2.0 sections.

Comment (zaki50) [inline tools/release.sh]: Ah, `-eq 1` is important. Got it.

**Pull request merged**